### PR TITLE
librados: block MgrClient::start_command until mgrmap

### PR DESCRIPTION
**Pull request by tchaikov**
Fixes: https://tracker.ceph.com/issues/23627

### CODE REVIEW

**Comment by tchaikov** [comment]
this PR tries to address the test failure we have when testing #21698

File: src/ceph.in [hunk 1]
```diff
@@ -134,10 +134,19 @@ import subprocess
 from ceph_argparse import \
     concise_sig, descsort_key, parse_json_funcsigs, \
     matchnum, validate_command, find_cmd_target, \
-    send_command, json_command, run_in_thread
+    json_command, run_in_thread
 
 from ceph_daemon import admin_socket, DaemonWatcher, Termsize
 
+try:
+    errno.ETIMEOUT
```
Comment (jcsp) [inline src/ceph.in]: On my system there's an `errno.ETIMEDOUT` (with the D in the middle)

Comment (tchaikov) [reply src/ceph.in]: no wonder i need to monkey patch errno here. but in my ipython, errno.E<TAB> always shows me something!

**Comment by tchaikov** [comment]
- http://pulpito.ceph.com/kchai-2018-05-04_16:50:59-rados-wip-kefu-testing-2018-05-04-2301-distro-basic-smithi/
- http://pulpito.ceph.com/kchai-2018-05-05_02:02:06-rados-wip-kefu-testing-2018-05-04-2301-distro-basic-mira/

**Pull request merged**